package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "testcafe-vue-selectors",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
do we have a breaking change here? If yes, what was changed? /cc @LavrovArtem @miherlosev